README.md

```diff
@@ -96,6 +96,64 @@ class Test2 {
 } 
 ```
 
+#### Using custom inspections
+
+Scapegoat supports custom inspections, for example to enforce a
+project-specific rule.
+
+How to write and test the inspection class is described in [the scalac-scapegoat-plugin CustomInspections readme](https://github.com/sksamuel/scalac-scapegoat-plugin/blob/master/CustomInspections.md)
+
+Your inspections need to be compiled before the code in the rest of your project is compiled (like macros). You can achieve this with a project set up like the following:
+
+    object MyBuild extends Build {
+    
+      val scalaV = "2.11.6"
+    
+      lazy val project = Project("MyProject", file("."))
+        // "aggregate" causes these child projects to be built and tested whenever
+        // the parent project is built and tested.
+        .aggregate(inspections)
+        .settings(
+          scalaVersion := scalaV,    
+          // You must list here all the custom inspections by their fully-qualified class name:
+          scapegoatCustomInspections := List(
+            "my.inspections.InspectionOne",
+            "my.inspections.InspectionTwo"),
+          scapegoatCustomInspectionsClasspath := List(
+            (classDirectory in inspections in Scapegoat).value))
+        .dependsOn(inspections)
+    
+      lazy val inspections = Project("inspections", file("inspections"))
+        .settings(
+          libraryDependencies ++= List(
+            "com.sksamuel.scapegoat" %% "scalac-scapegoat-plugin" % "1.1.0",
+            "org.scala-lang" % "scala-compiler" % scalaV),
+          scalaVersion := scalaV)
+    }
+
+
+If your custom inspections come from a separate JAR, then your build will need to look more like this:
+
+    object MyBuild extends Build {
+    
+      val scalaV = "2.11.6"
+    
+      lazy val project = Project("MyProject", file("."))
+        .settings(
+          scalaVersion := scalaV,    
+          // You must list here all the custom inspections by their fully-qualified class name:
+          scapegoatCustomInspections := List(
+            "my.inspections.InspectionOne",
+            "my.inspections.InspectionTwo"),
+          scapegoatCustomInspectionsClasspath := List(
+            "lib/my-inspections-1.0.0.jar")
+    }
+
+The `scapegoatCustomInspectionsClasspath` SBT SettingKey needs to list the classpath entries at which your inspection classes can be loaded. (There is not currently automatic support for using an Ivy-provided JAR dependency here. You may be able to do this by accessing the "update" SBT TaskKey. There is some code along these lines in [sbt-scapegoat](https://github.com/sksamuel/sbt-scapegoat/blob/ae4231d1341eeece323e111c757d57d904e66f7b/src/main/scala/com/sksamuel/scapegoat/sbt/ScapegoatSbtPlugin.scala#L41) which you can use for inspiration. However, this probably won't work as SBT Settings may not depend on SBT Tasks.)
```

Comment:
Do you have any idea how I can work around this?
I haven't really tested the external-JAR version yet, so I haven't looked into this. I will do soon.

(I have tested the in-project version)